nyaa/static/js/main.js

```diff
@@ -188,7 +188,47 @@ document.addEventListener("DOMContentLoaded", function() {
 	}
 });
 
-//
```

Comment:
I suggest that you **_do_** add at least a one-liner to describe what the following lines do, like most of the functions above.